fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1223,8 +1219,6 @@ public void loadImage(String imageDir) throws IOException, DdlException {
             checksum = loadInsertOverwriteJobs(dis, checksum);
             checksum = nodeMgr.loadComputeNodes(dis, checksum);
             remoteChecksum = dis.readLong();
-            checksum = loadShardManager(dis, checksum);
-            remoteChecksum = dis.readLong();
```

Comment:
Should not remove the original load and save shard manager logic, because this will case checkpoint compatibility issues.